rfhub2/cli/keywords_importer.py

```diff
@@ -129,7 +133,9 @@ def _traverse_paths(self, path: Path) -> Set[Path]:
                     valid_lib_paths.add(item)
         return valid_lib_paths
 
-    def create_collections(self, paths: Set[Path]) -> List[Dict]:
+    def create_collections(
+        self, paths: Set[Path]
+    ) -> List[Tuple[Collection, List[KeywordUpdate]]]:
```

Comment:
This tuple type appears a lot in the logic. Maybe you could define a dataclass with these two fields so you have clearer signatures and can access named fields instead of indexed items of a tuple

Reply:
Should I put dataclass in `rfhub2/db/repository/keyword_statistics_repository.py` or somewhere else?

Reply:
@MaciejWiczk as far as I can see it will be used only in this module so you can add it here

Reply:
Done. I cherry-picked commit for fixing acceptance tests, to check pipeline.